webapp/src/services/account-service.ts

```diff
@@ -15,8 +17,9 @@ const generateId = (): string => {
 export const AccountService = {
   async login(phone: string, password: string) {
     const args = { phone, password };
-    const res = await axios.post<User>(`${API_URL}/login`, args);
-    return res.data;
+    const res = await axios.post<LoginResult>(`${API_URL}/users/login`, args);
+    Auth.setAccessToken(res.data.token._id);
+    return res.data.user;
   },
   async donate(accountId: string, amount: number) {
```

Comment:
Do we support this `donate` endpoint on the server? I don't believe we do. Please remove all methods/endpoints that we don't implement.